frontend/src/metabase/components/SaveQuestionModal.css

```diff
@@ -0,0 +1,23 @@
+:local(.saveQuestionModalFields) {
+    overflow: hidden;
+}
+
+.saveQuestionModalFields {
+    overflow: hidden;
```

Comment:
Looks like .saveQuestionModalFields is declared twice here. It'd be nice if we didn't have to duplicate these and or the very least did something like
```css
.saveQuestionModalFields,
:local(.saveQuestionModalFields) {
  overflow: hidden;
}
```